pkg/reconciler/v1alpha1/taskrun/taskrun_test.go

```diff
@@ -82,6 +82,13 @@ var (
 
 	saTask = tb.Task("test-with-sa", "foo", tb.TaskSpec(tb.Step("sa-step", "foo", tb.Command("/mycmd"))))
 
+	taskEnvTask = tb.Task("test-task-env", "foo", tb.TaskSpec(
+		tb.TaskEnvVar("FRUIT", "APPLE"),
```

Comment:
maybe include one we don't override as well?

Reply:
Latest iteration shows this working right - it shows up in the init containers and the nop container.